libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/guidance/GuidanceViewUrlMatcher.kt

```diff
@@ -0,0 +1,32 @@
+@file:JvmName("GuidanceViewUrlMatcher")
+
+package com.mapbox.navigation.ui.maps.guidance
+
+import com.mapbox.api.directions.v5.models.BannerComponents
+
+internal fun String.guidanceViewUrlMatcher(
+    type: String,
+    accessToken: String,
+    key: String = "access_token=",
+): String {
+    when (type) {
+        BannerComponents.JCT -> {
```

Comment:
Why a different case for `JCT`? Logic for appending a query parameter should be universal for all and any URLs.

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/6239#discussion_r964746867